[2022-12-11T08:10:56.212+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: etl_solar.checkfiles.check_weather_table manual__2022-12-11T08:10:48.533274+00:00 [queued]>
[2022-12-11T08:10:56.339+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: etl_solar.checkfiles.check_weather_table manual__2022-12-11T08:10:48.533274+00:00 [queued]>
[2022-12-11T08:10:56.348+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-12-11T08:10:56.354+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-12-11T08:10:56.361+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-12-11T08:10:56.501+0000] {taskinstance.py:1383} INFO - Executing <Task(MySqlOperator): checkfiles.check_weather_table> on 2022-12-11 08:10:48.533274+00:00
[2022-12-11T08:10:56.523+0000] {standard_task_runner.py:55} INFO - Started process 148 to run task
[2022-12-11T08:10:56.556+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'etl_solar', 'checkfiles.check_weather_table', 'manual__2022-12-11T08:10:48.533274+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocess.py', '--cfg-path', '/tmp/tmpabn7xy23']
[2022-12-11T08:10:56.567+0000] {standard_task_runner.py:83} INFO - Job 33: Subtask checkfiles.check_weather_table
[2022-12-11T08:10:57.072+0000] {task_command.py:376} INFO - Running <TaskInstance: etl_solar.checkfiles.check_weather_table manual__2022-12-11T08:10:48.533274+00:00 [running]> on host 01c108b234e9
[2022-12-11T08:10:58.085+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_solar
AIRFLOW_CTX_TASK_ID=checkfiles.check_weather_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T08:10:48.533274+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T08:10:48.533274+00:00
[2022-12-11T08:10:58.091+0000] {mysql.py:84} INFO - Executing: CREATE TABLE IF NOT EXISTS weather_info(
                locdatetime VARCHAR(40),
                location VARCHAR(40),
                updated_date VARCHAR(13),
                temperature FLOAT,
                precipitation_form FLOAT,
                precipitation_prob FLOAT,
                humidity FLOAT,
                wind_speed FLOAT,
                wind_direction FLOAT,
                cloud FLOAT,
                primary key(locdatetime,location))
[2022-12-11T08:10:58.202+0000] {base.py:71} INFO - Using connection ID 'mysql_solar' for task execution.
[2022-12-11T08:10:58.400+0000] {sql.py:315} INFO - Running statement: CREATE TABLE IF NOT EXISTS weather_info(
                locdatetime VARCHAR(40),
                location VARCHAR(40),
                updated_date VARCHAR(13),
                temperature FLOAT,
                precipitation_form FLOAT,
                precipitation_prob FLOAT,
                humidity FLOAT,
                wind_speed FLOAT,
                wind_direction FLOAT,
                cloud FLOAT,
                primary key(locdatetime,location)), parameters: None
[2022-12-11T08:10:58.418+0000] {sql.py:324} INFO - Rows affected: 0
[2022-12-11T08:10:58.535+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=etl_solar, task_id=checkfiles.check_weather_table, execution_date=20221211T081048, start_date=20221211T081056, end_date=20221211T081058
[2022-12-11T08:10:58.692+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-12-11T08:10:59.085+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
